src/Main.js

```diff
@@ -1,16 +1,40 @@
 import { Component } from 'react';
+import Data from './data.json';
```

Comment:
Great work on the lab!

For this line, change `Data` to lowercase `data` since the json data isn't a component